.github/workflows/publish-release.yml

```
@@ -202,7 +202,7 @@ jobs:
   build-test:
     needs:
       - check_branch
-    runs-on: buildjet-4vcpu-ubuntu-2204
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
I would prefer to explicitly select a version.